### ISSUE TITLE
Docker for version 7

### ISSUE THREAD

**Issue by ezanattatray** [directus/directus-docker#49]
Currently, docker only has the option of version 6, would have a forecast of the availability of the new version?

**Comment by benhaynes** [comment]
`beta.4` will be released in a few hours, next week is `rc.1`, then we'll either have `rc.2` or the full release... depending on community feedback. We'll be creating a Docker container with the proper release... and perhaps for the RC.

**Comment by ezanattatray** [comment]
Thank you for this information @benhaynes

**Comment by simofacc** [comment]
@benhaynes Is there a combined version that sets up the API and the APP like it happens now with the 6.4 docker compose file?

**Comment by k15a** [comment]
@simofacc There is one here: https://github.com/directus/directus-docker/tree/v7 but it seems that it's not working correctly at least for me. Maybe you can check it out and fix it.

**Comment by benhaynes** [comment]
Version 7 has a bunch of ways you can use Docker. You can install the App/API individually with it (each repo has a Dockerfile), or install the bundle (what you're looking for) on the combined-build v7 branch:

https://github.com/directus/directus/tree/build-v7

No need for a dedicated Docker repo after we release v7!

**Comment by simofacc** [comment]
@benhaynes looks like the app is missing from the docker installation. The API and DB are setup but the app is not within that branch.

**Comment by benhaynes** [comment]
Yeah, I read into that issue yesterday too. It's been fixed in the API (a commit yesterday) and will be pushed in the next release.

**Comment by k15a** [comment]
Using the https://github.com/directus/directus/tree/build-v7 works for me. You only have to clone it and run `docker-compose up`.

**Comment by Marcnuth** [comment]
Thanks, @k15a. 

Finally, I used the separated repositories: app & api to build latest docker images.

**Comment by lrkwz** [comment]
It isn't available through `https://hub.docker.com/r/getdirectus/directus/` ?

**Comment by benhaynes** [comment]
I believe that link is for v6, once we have everything looking good for v7 we'll be updating our docker hub... in fact our new Hub Organization name is `directus` (which we'll use for v7+).
